aztk/utils.py

```diff
@@ -0,0 +1,210 @@
+import getpass
```

Comment:
alpha sort?

Reply:
Pylint complains if you don't do standard lib imports first.